### PR TITLE
Emit stderr output from `npm` on bootstrap failure

### LINKED ISSUE #247: bootstrap fails silently
With #217 being merged, `lerna bootstrap` now silently fails without giving a reason.
```
➜  npmE lerna bootstrap
Lerna v2.0.0-beta.22
Independent Versioning Mode
Linking all dependencies
Errored while running BootstrapCommand.execute
1
```
instead of:
```
Lerna v2.0.0-beta.22
Independent Versioning Mode
Linking all dependencies
normalizeTree             ▄ ╢░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░╟╟
npm ERR! Darwin 15.5.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "xxx@^0.0.0" "skatejs@1.0.0-beta.2"
npm ERR! node v6.2.1
npm ERR! npm  v3.9.3
npm ERR! code E404

npm ERR! 404 Registry returned 404 for GET on https://registry.npmjs.org/xxx
```

with
```javascript
//  stdio: "ignore"
```
in https://github.com/lerna/lerna/pull/217/commits/86bb31e46d1012f82b76043350db4d83f7701844#diff-5463599b8d45d087bae97f0d5df54fdaR15

### PR DESCRIPTION
**Pull request by gigabo**
This fixes #247.

### CODE REVIEW

File: src/ChildProcessUtilities.js [hunk 1]
```diff
@@ -33,8 +33,21 @@ export default class ChildProcessUtilities {
   }
 
   static spawn(command, args, opts, callback) {
-    child.spawn(command, args, objectAssign({
+    let stderr = "";
+
+    const cp = child.spawn(command, args, objectAssign({
```
Comment (thejameskyle) [inline src/ChildProcessUtilities.js]: Can you call it something other than `cp`

Comment (gigabo) [inline src/ChildProcessUtilities.js]: Done.  Now `childProcess`.

**Comment by joscha** [comment]
Is there anything missing before we can merge this?

**Comment by joscha** [comment]
awesome @gigabo - is there a way you can publish a new beta version that contains this fix?

**Pull request merged**